### PR TITLE
contrib/inotify-tools: new package

### PR DESCRIPTION
**Pull request by c7skasku**

### CODE REVIEW

File: contrib/inotify-tools/template.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+pkgname = "inotify-tools"
+pkgver = "4.23.9.0"
+pkgrel = 0
+build_style = "gnu_configure"
+makedepends = ["automake", "libtool"]
```
Comment (nekopsykose) [inline contrib/inotify-tools/template.py:5]: does it work if you put slibtool here?

File: contrib/inotify-tools/template.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+pkgdesc = "Command-line interface to inotify"
+maintainer = "c7s <c7s@kasku.net>"
+license = "GPL-2.0-only"
+url = "https://github.com/inotify-tools/inotify-tools"
+source = f"https://github.com/inotify-tools/inotify-tools/archive/refs/tags/{pkgver}.tar.gz"
```
Comment (nekopsykose) [inline contrib/inotify-tools/template.py:10]:
```suggestion
source = f"{url}/archive/refs/tags/{pkgver}.tar.gz"
```

File: contrib/inotify-tools/template.py (hunk 1 above)
Comment (nekopsykose) [reply contrib/inotify-tools/template.py:5]: yeah

**Comment by nekopsykose** [comment]
> btw I don't know how the symlinks are meant to look, does that all look fine?

you only have to run `./cbuild relink-subpkgs`

File: contrib/inotify-tools/inotify-tools-devel-static [hunk 3]
```diff
@@ -0,0 +1 @@
+inotify-tools-devel-static
```
Comment (nekopsykose) [inline contrib/inotify-tools/inotify-tools-devel-static:1]: this and the symlink above are inside the inotify-tools folder and so don't actually do anything (and are merely ignored by all the tooling, so there's no errors and also nothing happens, but they shouldn't be there). a funny failure i haven't seen before i guess :D  

in general you never have to make these, just running cbuild relink-subpkgs does everything for you

File: contrib/inotify-tools/template.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+pkgdesc = "Command-line interface to inotify"
+maintainer = "c7s <c7s@kasku.net>"
+license = "GPL-2.0-only"
+url = "https://github.com/inotify-tools/inotify-tools"
+source = f"{url}/archive/refs/tags/{pkgver}.tar.gz"
+sha256 = "1dfa33f80b6797ce2f6c01f454fd486d30be4dca1b0c5c2ea9ba3c30a5c39855"
+
+
+def post_install(self):
+    self.install_license("COPYING")
```
Comment (JamiKettunen) [inline contrib/inotify-tools/template.py:15]: This wasn't needed for `GPL-2.0-only` btw, it fails to pkg when one isn't installed but should be

**Pull request merged**